UX/ui-service/Common/Tags/design.md

```diff
@@ -0,0 +1,45 @@
+# Tags - Common Component
+* Tags are used in multiple areas throughout the UI.
+* Tags are made up of two parts, a category and value. Some categories allow for more than one value to be associated.  
+* Tags can be applied to a single item or multiple items at one time depending on the use case.
+* In all cases, the Tag selection dropdowns and the tag labels should be visually consistent.
```

Comment:
how about something like "In all cases, the tag selection mechanism should be visually consistent." ?